### PR TITLE
UI fixes 3

### PR DESCRIPTION
**Pull request by tinashechiraya**
https://github.com/kartoza/miniSASS/assets/70011086/d9a54ba1-fc3e-4296-901c-2262bd8ce775

contact_back

completed_reg

Description:
when the site is not ready a loading bar will appear
I have also moved the success message from the contact us form to appear on its own because it overlapping the fields in the form
I have also updated the completed registration modal styling so it matches the universal theme for the website

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 48 sec

**Review by dimasciput — APPROVED** [review]

File: django_project/minisass_frontend/src/components/ContactFormModal/index.tsx [hunk 1]
```diff
@@ -38,31 +40,35 @@ const ContactFormModal: React.FC<ContactFormModalProps> = ({ isOpen, onClose, on
       phone: '',
       message: '',
     });
-  };;
+  };
+
+  const [SendingMessage, setSendingMessage] = useState(false);
+  const [MessageSent, setMessageSent] = useState(false);
```
Comment (dimasciput) [inline django_project/minisass_frontend/src/components/ContactFormModal/index.tsx]:
```suggestion
  const [sendingMessage, setSendingMessage] = useState(false);
  const [messageSent, setMessageSent] = useState(false);
```

File: django_project/minisass_frontend/src/components/ContactFormModal/index.tsx [hunk 2]
```diff
@@ -98,15 +108,50 @@ const ContactFormModal: React.FC<ContactFormModalProps> = ({ isOpen, onClose, on
...
           background: 'white',
           padding: 0,
         },
       }}
     >
-      {isOpen && (
+
+      {SendingMessage ? (
+        <LinearProgress color="success" />
+      ) : MessageSent ? (
```
Comment (dimasciput) [inline django_project/minisass_frontend/src/components/ContactFormModal/index.tsx]:
```suggestion
      {sendingMessage ? (
        <LinearProgress color="success" />
      ) : messageSent ? (
```

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 55 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 17 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 16 sec

**Pull request merged**